gkr_iop/src/selector.rs

```diff
@@ -170,10 +180,69 @@ impl<E: ExtensionField> SelectorType<E> {
                     });
                 Some(sel.into_mle())
             }
+            // also see evaluate() function for more explanation
+            SelectorType::QuarkBinaryTreeLessThan(_) => {
+                assert_eq!(ctx.offset, 0);
+                // num_instances: number of prefix one in leaf layer
+                let mut sel: Vec<E> = build_eq_x_r_vec(out_point);
+                let n = sel.len();
+
+                let num_instances_sequence = (0..out_point.len())
+                    // clean up sig bits
+                    .scan(
+                        (ctx.num_instances / 2, ctx.num_instances.div_ceil(2)),
```

Comment:
The initial state is wrong. I think it should be `(ctx.num_instances, ctx.num_instances)`.

Reply:
selector 
- determine op(`v[2*i]`. `v[2*i+1]`) op is `sel_add` or `1 - sel_add = sel_bypass_left`
- thus selector length is only halve size of overall leaf vector

Reply:
I get what you mean. The number of nonzero entries in the `sel_add` is the number of ec point additions need to be done.